.pre-commit-config.yaml

```
@@ -48,5 +48,8 @@ repos:
     -   id: pylint
         name: pylint
         entry: pylint
+        args:
+        -   --load-plugins=pylint_django
```

The spacing is weird, it should be smth like
```suggestion
        - --load-plugins=pylint_django
        - --good-names=urlpatterns
```